components/eamxx/scripts/jenkins/jenkins_common_impl.sh

```diff
@@ -108,7 +108,7 @@ if [ $skip_testing -eq 0 ]; then
 
       # Add a memcheck test for mappy/weaver for nightlies
       if [[ "$SCREAM_MACHINE" == "mappy" || "$SCREAM_MACHINE" == "weaver" ]]; then
-        ./scripts/gather-all-data "./scripts/test-all-scream -t dbg --mem-check ${TAS_ARGS} -c SCREAM_TEST_SIZE=SHORT" -l -m $SCREAM_MACHINE
+        ./scripts/gather-all-data "./scripts/test-all-scream -t mem ${TAS_ARGS}" -l -m $SCREAM_MACHINE
```

Comment:
Should we also add builds on weaver for CMS, CMR, CMI?

Reply:
Good question. I was wondering that too.

Reply:
It looks like our current memcheck build on weaver takes 45-60min. Assuming similar weight for the other 3, and the fact that weaver is not very used at night (I think), we may be able to afford it.

Reply:
Also, we may launch the 4 compute-sanitizer builds in ||, since there are 4 gpus on each weaver node...

Reply:
Nice, so we can do 4 for the price of 1? Seems like an obvious win although it might have us chasing down false positives all the time.

Reply:
Well, not exactly. When running a single build, we can use 4 gpus as well, so running 4 builds we make each of them slower. The build phase will however be slightly faster (IIRC, most of the build time on weaver is spent on a couple of files, shoc/p3_functions_f90.cpp). I guess we'd have to try it, to see how expensive they really are...

Reply:
Regarding false positives, that might be true, but what's the point of having them if we don't run them? I would say "let's activate them", and if we see that they hit too many false positives, we can remove them from our nightlies (leaving them as an option, to debug stuff).